frontend/src/app/[locale]/opportunity/[id]/page.tsx

```diff
@@ -0,0 +1,64 @@
+import { Metadata } from "next";
+import NotFound from "../../../not-found";
+import OpportunityListingAPI from "../../../api/OpportunityListingAPI";
+import { getTranslations } from "next-intl/server";
+
+export async function generateMetadata() {
+  const t = await getTranslations({ locale: "en" });
+  const meta: Metadata = {
+    title: t("OpportunityListing.page_title"),
+    description: t("OpportunityListing.meta_description"),
+  };
+  return meta;
+}
+
+export default async function OpportunityListing({
```

Comment:
I'm still catching up on whether or not making a page async affects ability to statically render the page. The opportunity pages are the same for everyone and rarely update so should not be rebuilt for every single request. Statically rendering seems like the easiest way to achieve that.

Reply:
We can punt this consideration for #85 .

Reply:
🧵  here https://nava.slack.com/archives/C057K146W8H/p1718912909890579